src/NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj

```diff
@@ -69,9 +69,9 @@ Supported platforms:
     <PackageReference Include="Microsoft.AspNetCore.Routing.Abstractions" Version="1.1.2" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net461' ">
-    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Routing.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Routing.Abstractions" Version="2.1.1" />
```

Comment:
Maybe we should state this in the readme (and/or add a comment here) so that others (including me) don't forget it ;)

Reply:
Well not the master of the universe. But usually sub-components at the bottom of the pyramid should be the slowest at updating to newer versions. Not a big fan for dependency-update-robots :)

Reply:
I've created a new PR so this is more explicit/transparent. https://github.com/NLog/NLog.Web/pull/476